### PR TITLE
correct helm documentation

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by arihantg** [comment]
skip jira-id-check

**Comment by arihantg** [comment]
build amko

File: README.md [hunk 1]
```diff
@@ -66,40 +66,40 @@ Following steps have to be executed on all member clusters:
...
-   NAME     	CHART VERSION    	APP VERSION      	DESCRIPTION
-   amko/amko	1.10.1	            1.10.1	            A helm chart for Avi Multicluster Kubernetes Operator
-
+   $ helm show chart oci://projects.registry.vmware.com/ako/helm-charts/amko --version 1.10.1
+
+   Pulled: projects.registry.vmware.com/ako/helm-charts/amko:1.10.1
+   Digest: sha256:xyxyxxyxyx
+   apiVersion: v2
+   appVersion: 1.10.1
+   description: A helm chart for Avi Kubernetes Operator
```
Comment (swathinsankaran) [inline README.md:79]: we should check where this description is coming from and change it.

**Review by swathinsankaran — APPROVED** [review]
LGTM

**Pull request merged**